docs/src/examples/components/Portal/Types/PortalExampleControlled.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
```

Comment:
No typescript in examples please.

Reply:
I'd rather allow both, the similar way as it is done on the Android Developer docs
![image](https://user-images.githubusercontent.com/5530260/44721979-e2812e80-aacb-11e8-9ef0-e8282c6f1325.png)